holes/views.py

```diff
@@ -1,22 +1,23 @@
-from rest_framework.views import APIView
+from rest_framework import mixins, status, exceptions, views, generics
```

Comment:
To rozwiązanie z importami podoba mi się bardziej od poprzedniego. Pomijając że nie trzeba dopisywać kolejnych klas do importu to w kodzie lepiej widać co się skąd bierze ...